python/sglang/srt/model_executor/cuda_graph_runner.py

```diff
@@ -49,13 +49,16 @@
     empty_context,
     get_available_gpu_memory,
     get_device_memory_capacity,
+    is_npu,
     rank0_log,
     require_attn_tp_gather,
     require_gathered_buffer,
     require_mlp_sync,
     require_mlp_tp_gather,
 )
 
+_is_npu = is_npu()
```

Comment:
We should not modify cuda_graph_runner.py because we will never use it in npu_graph mode